tools/ci/azure/install_chrome.yml

```diff
@@ -1,11 +1,6 @@
 steps:
-# The conflicting google-chrome and chromedriver casks are first uninstalled.
-# The raw google-chrome@dev.rb cask URL is used to bypass caching.
 - script: |
     set -eux -o pipefail
-    HOMEBREW_NO_AUTO_UPDATE=1 brew uninstall --cask google-chrome || true
-    HOMEBREW_NO_AUTO_UPDATE=1 brew uninstall --cask chromedriver || true
```

Comment:
Did we want to keep these lines?

Reply:
This shouldn't make any difference; I don't see why uninstalling Google Chrome (stable) would make any difference. `chromedriver` there's no reason why uninstalling and re-installing would change anything?